lib/api_client/execute_request.js

```diff
@@ -66,6 +66,14 @@ function execute_request(method, params, auth, api_url, callback, options = {})
     request_options.headers['Content-Length'] = Buffer.byteLength(query_params);
   }
   handle_response = function (res) {
+    const {hide_sensitive = false} = config();
+    const sanitized_request_options = {...request_options};
```

Comment:
I know the previous line suggests something different, but the convention in js is to use lower camel case for naming the variables, in this case it'd be: `sanitizedRequestOptions`. We can also make it shorter to `sanitizedOptions` because it's clear what those options are from the context where they are used.